src/Components/Aspire.Qdrant.Client/AspireQdrantExtensions.cs

```diff
@@ -80,6 +82,41 @@ private static void AddQdrant(
             builder.Services.AddKeyedSingleton(serviceKey, (sp, key) => ConfigureQdrant(sp));
         }
 
+        if (!settings.DisableHealthChecks)
+        {
+            // use http client for register health check
+            var httpClientName = serviceKey is null ? "qdrant-healthchecks" : $"qdrant-healthchecks_{connectionName}";
```

Comment:
Why not implement this in https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/issues/2186? Then everyone can use it, and not just people using the Aspire integration.